app-admin/pinkybar/pinkybar-9999.ebuild

```diff
@@ -0,0 +1,135 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+EGIT_REPO_URI="https://github.com/su8/pinky-bar.git"
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit git-r3 python-single-r1 autotools
+
+DESCRIPTION="Gather some system information and show it in this statusbar program"
+HOMEPAGE="https://github.com/su8/pinky-bar"
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="+alsa colors drivetemp drivetemp-light dvd dwm ip keyboard libnl lisp lua
+	mouse mpd ncurses numcapslock +net +pci perl python2 r ruby sensors slang
+	smartemp tcl"
+
+DEPEND="
+	sys-devel/autoconf
```

Comment:
`autotools.eclass` takes care of depending on auto* and related dependencies for you.